src/Microsoft.OData.Edm/Schema/CoreModel/EdmCoreModelComplexType.cs

```diff
@@ -0,0 +1,86 @@
+//---------------------------------------------------------------------
+// <copyright file="EdmCoreModelComplexType.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.OData.Edm
+{
+    /// <summary>
+    /// The built-in Edm.ComplexType abstrat type in the core model.

```

Comment:
>abstrat [](start = 37, length = 7)

nit: spelling "abstract"

Reply:
Thanks, also modified for `EdmCoreModelEntityType, EdmCoreModelPathType`